apps/loan/src/entities/vaults.ts

```diff
@@ -0,0 +1,106 @@
+import { queryFactory } from '@/shared/model/query'
+import { createValidationSuite } from '@/shared/lib/validation'
+
+export interface AmmBalances {
```

Comment:
They're essentially the same, but any reason why you went with interfaces instead of types like you did in `crvusd.ts` and `lending.ts` in the same folder?

Reply:
Types are fine. These were generated